### ISSUE TITLE
All accesses from/to ip address are blocked

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#49]
```
imported trac ticket
created: 2009-10-24 00:49:13
reporter: emk
```

Just a simple typo, but critical.
https://www.requestpolicy.com/dev/browser/src/modules/DomainUtil.jsm#L144
144     if (e.name == "NS_ERROR_HOST_IS_IP_ADDRESS ") {
Please trim an extra trailing space.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2009-10-24 07:43:57
author: justin
```

Thanks for catching this. I've just introduced this bug recently, I believe, while fixing a bug in the same part of the code. I'm walking out the door right now to get on flight so can't fix it right now. In the mean time, I've pulled the 0.5.11 version's file from AMO so that it doesn't get reviewed/approved while I'm out of town for the weekend.

I fix this first thing next week and push 0.5.12.

Thanks again for finding this!
